dockers/docker-orchagent-cavm/copp.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
It is moved to https://github.com/Azure/sonic-swss/blob/master/swssconfig/sample/00-copp.config.json